### ISSUE TITLE
Thorough questions about designSize property

### ISSUE THREAD

**Issue by Ruslanbek0809** [OpenFlutter/flutter_screenutil#278]
Hi. First of all thank you for your contribution. I have few questions about `designSize`. I thoroughly checked all questions about `designSize` and documentation itself. But I still have questions about it. 
1. Should I change `designSize` from UI Design Draft `Size(360, 690)` to actual device size(in my case `Size(392,834)`) when I start working on my device during development process in order to be sure that it is gonna be responsive in all devices?
2. Is it ok to use `ScreenUtil().setWidth()` in all paddings/margins/height/width on my app when I set `designSize` to  UI Design Draft or actual device size? I mean will all these paddings/margins/height/width still be responsive when I test app responsiveness on Ipad or Tablet or any kind of phones?
3. What if I change my phone to another large phone in development process? Do i have to change again `designSize` to my new phone's actual device size?
4. I don't want to use `sw` or `sh` in my paddings/margins/height/width. Instead I want to use `ScreenUtil().setHeight()` and `ScreenUtil().setWidth()` in all these. Will it still be responsive in all devices?

**Comment by lizhuoyuan** [comment]
1. no
2. yes
3. no
4. yes

designSize represents the screen size in the UI design draft, it will not change.
If there is no UI design draft, please refer to the first mobile phone developed, but it still does not need to be modified.

**Comment by Ruslanbek0809** [comment]
Thanks for clarification. But still when I use default UI Design Draft `Size(360, 690)`, I see that when I set `ScreenUtil().setWidth(200)` and `ScreenUtil().setHeight(200)`, they are not in exact 200 but 218 and 242. That part confused me. Because when I set paddings/margins/height/width to `ScreenUtil().setWidth()` and `ScreenUtil().setHeight()`, I don't want them to act different in size when i check in other real devices during development. Are these gonna act normal or different? It is the last question i am about to ask

**Comment by lizhuoyuan** [comment]
ScreenUtil().setWidth(200) != 200   ✅
To ensure the display effect of different screens.
If you want a size that is always the same on different screens, you can write the value directly.

**Comment by Ruslanbek0809** [comment]
Okay now I got it. So even if I set default UI Design Draft Size(360, 690) like this and use `ScreenUtil().setWidth()` and `ScreenUtil().setHeight()` for paddings/margins/height/width, it DOES change according to different device sizes. As a new user of this great package I got little confused bc this term "Design Draft". I thought how the heck it gets responsive if we give initial `designSize` value. It is clear for me now. Special thanks for your quick explanations.
